scripts/mg_net/README.md

```diff
@@ -38,28 +46,30 @@ optional arguments:
                         14. Replication Initiation
                         15. Cellular Division
                           
-  -r <min> <max>        Specify the minimum and maximum value for the "discrete uniform" distribution 
+  `-r <min> <max>`:        Specify the minimum and maximum value for the "discrete uniform" distribution 
                         for the random initialization of species initial count (default from 10 to 1000)
-  -n <mean> <range_value_from_mean>
+
+  `-n <mean> <range_value_from_mean>`:
                         Specify the mean and the range value from mean for the normal distribution 
```

Comment:
Done

Reply:
Can we change this option to <min> <max> as well? If so, we can define a normal distribution with mean = (min+max)/2 and stdev = (max-mean/)3. Also, make sure to check if the min given by a user is a positive number.
As I understand, this will still allow some random values less than min as small as -inf or greater than max as large as +inf. We need to check the random value every time it is pulled from the distribution, and bound it to the interval between min and max.

Reply:
Hi Jae-Seung,

I have made all the changes as you suggested. Please let me know if you need anything else.
Thank you.

Kind regards,
Konstantia